doc/help.md

```diff
@@ -889,6 +889,86 @@ JOB:
 
 ### External
 
+<h4 id="[">[</h4>
```

Comment:
Nothing will point here, so I think we should combine it with `test`.   The list of topics is in the index doc:

https://www.oilshell.org/release/0.8.pre10/doc/help-index.html